lib/context_adapter_helper.js

```diff
@@ -23,7 +23,9 @@
 
 'use strict';
 
+var caLogger = require('logops');
 var caConfig = require('./context_adapter_configuration');
+caLogger.setLevel(caConfig.LOGOPS_LEVEL);
```

Comment:
I mean, something such as:
```javascript
var caConfig = require('./context_adapter_configuration'),
      caLogger = require('logops');
caLogger.setLevel(caConfig.LOGOPS_LEVEL);

// Since these modules also need to log messages... The other option would be to include functions for logging in the main module and pass it to the other ones :p
var caHelper = require('./context_adapter_helper')(caLogger),
      caServer = require('./context_adapter_server')(caLogger);
```